src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -63,11 +63,20 @@ namespace winrt::TerminalApp::implementation
         _tabView = _tabRow.TabView();
         _rearranging = false;
 
-        // GH#3581 - There's a platform limitation that causes us to crash when we rearrange tabs.
-        // Xaml tries to send a drag visual (to wit: a screenshot) to the drag hosting process,
-        // but that process is running at a different IL than us.
-        // For now, we're disabling elevated drag.
-        const auto isElevated = ::winrt::Windows::UI::Xaml::Application::Current().as<::winrt::TerminalApp::App>().Logic().IsElevated();
+        // GH#2455 - Make sure to try/catch calls to Application::Current,
```

Comment:
ah, sorry :smile:

Reply:
lol np, this wasn't ever tested before so it wasn't a problem 😄